File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashSet <Integer> checkUp = new HashSet<Integer>(); 

        for (int num: nums)
        {
            if (checkUp.contains(num))
            {
                return true; 
            }
            else 
            {
                checkUp.add(num);
            }
        }
        
        return false; 
        
    }
}

File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        thresHold = len(nums) / 2
        myMap = {}

        for i , num in enumerate(nums):
            if num not in myMap: 
                myMap[num] = 1

            else: 
                myMap[num] += 1


        for key in myMap: 

            if myMap[key] > thresHold: 
                return key